### ISSUE TITLE
No focus set on "No reporting yet in ...." screem with Talkback on

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#734]
Samsung Galaxy J3 Prime
Android version 7.0

Samsung Galaxy S5
Android version 6.0

App version: 1.0 (53)

With Talkback on, from the "COVID alert is active" menu screen: 

- Press "Need an 8-digit number" button
- Press "Choose province or territory" button from "You skipped choosing province or territory" screen.
- Choose a province other than Ontario from "Province or territory" screen and close it. 
- Focus is not set on any element on "No reporting yet in ..." screen

Expected result
The focus should be set on the screen.

Screenshots
![province1](https://user-images.githubusercontent.com/12256067/87720382-dc110400-c782-11ea-8b93-4be7d0627da0.gif)
![province3](https://user-images.githubusercontent.com/12256067/87720393-e03d2180-c782-11ea-8bb1-2943de7fbb20.gif)
